libr/anal/esil.c

```diff
@@ -495,13 +495,13 @@ static int esil_internal_read(RAnalEsil *esil, const char *str, ut64 *num) {
 	case '$':
 		*num = esil->address;
 		break;
-	case 'z': //zero-flag
+/*	case 'z': //zero-flag
 		{
 			ut64 m = genmask (esil->lastsz - 1);
 			*num = (((ut64) esil->cur & m) == 0);
 		}
 		break;
-	case 'b': //borrow
+*/	case 'b': //borrow
```

Comment:
dont start lines with */ iif there's a case in there,

Reply:
this is a tmp thing, I will remove it later completly